semi_processed/index.html

```diff
@@ -40,10 +40,10 @@
             body { font: 14px "Lucida Grande", Helvetica, Arial, sans-serif; }
             td { font: 12px "Lucida Grande", Helvetica, Arial, sans-serif; }
         </style>
+        <title>Sandbox</title>

```

Comment:
Same comment as the one for `raw/index.html` file - no need to add a title in line 43 since a title for the page had already been declared in line 28. 

If the sonar bug you were trying to fix is `Add a "title" attribute to this <iframe> tag.`, you'd want to add a title attribute in lines 50-51.